SMART_ACCOUNT_AUTH.md

```
@@ -0,0 +1,281 @@
+  FOR SELECT USING (status = 'published');
+
+-- Writes: Service role only
+CREATE POLICY "Service role can insert" ON video_assets
+  FOR INSERT TO service_role WITH CHECK (true);
+```
+
+## Data Flow
+
+### Video Upload Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)